package.json

```diff
@@ -15,7 +15,9 @@
   },
   "devDependencies": {
     "ejs": "3.1.7",
-    "nodeunit": "^0.11.3"
+    "gren": "^0.0.1",
```

Comment:
I think if you look at the `package.json` from `node-saml`, you'll see that a different package is used: https://github.com/node-saml/node-saml/blob/f86252bcbaa4435159d121d2e342ec89f94ad183/package.json#L68

Reply:
whoops! fixed in amended commit